README.md

```diff
@@ -1,36 +1,232 @@
 
-The current Twilio Massive IoT Developer Kit, version "Alfa" is based on a Wio factor development board with numerous Grove connectors and a Narrowband capable ublox SARA-N410 cellular module.  The developer kit has thusfar been distributed at [SIGNAL](https://www.twilio.com/signal) and comes with Grove sensors – humidity, light, and ultrasonic distance.
+This Arduino compatible library is backed by a more generic cellular module abstraction layer, [Twilio Breakout Massive SDK](https://github.com/twilio/Breakout_Massive_SDK), which also has support for the Quectel BG96 NB-IoT module.  The goal of this module is to make packet based communications easier when dealing with these cellular modules.
 
-It is important to note that you must have a Narrowband SIM to utilize this kit, a standard LTE SIM will not be able to connect to the network.
+## Quickstart with AlfaKit.
+This section should provide you with a quick start if everything goes smoothly for you. Please refer to the [Detailed guides](#detailed-guides) section for more information.
 
-This Arduino compatible library is backed by a more generic cellular module abstraction layer, [Twilio Breakout Massive SDK](https://github.com/twilio/Breakout_Massive_SDK), which also has support for the Quectel BG96 NB-IoT module.  The goal of this module is to make packet based communications easier when dealing with these cellular modules.
+1. Add Wio LTE board to Arduino. Use the [instructions from Seeed Wiki](http://wiki.seeedstudio.com/Seeed_Arduino_Boards/) for guidance.
+2. Clone the SDK recursively to the directory where your Arduino libraries live:
+
+```
+# Under Linux
+git clone --recursive https://github.com/twilio/Breakout_Massive_SDK_Arduino ~/Arduino/libraries/Breakout_Massive_SDK_Arduino
+```
+
+Alternatively you can download one of the [releases](https://github.com/twilio/Breakout_Massive_SDK_Arduino/releases) and install it as a [zip library](https://www.arduino.cc/en/guide/libraries#toc4). **Never use Download Zip button under 'Clone or download' list, because some dependencies will be missing from the downloaded archive**.
+
+3. Install sensor support libraries.
+  * [Temperature and Humidity sensor](https://www.arduinolibraries.info/libraries/grove-temperature-and-humidity-sensor)
+  * [Ultrasonic range sensor](https://www.arduinolibraries.info/libraries/grove-ultrasonic-ranger)
+
+4. Start Arduino IDE and open the sample in `File > Examples > Breakout Massive SDK Arduino Library > AlfaKit > Sample.ino`. By default it will build a sample for Grove temperature and humidity sensor. Edit `Sample.ino` to enable different sensors.
+
+5. Edit `config.h` and `tls_credentials.h` to configure the connection to your MQTT broker.
```

Comment:
I would put a call-out here to the detailed guide below on configuring an MQTT broker just in case the user gets stuck here.